### PR TITLE
Add "save image" button

### PR DESCRIPTION
**Pull request by ili3p**
Add a button to the bar that saves the currently displayed image.

### CODE REVIEW

**Comment by ili3p** [comment]
This solves #27

File: static/panes.js [hunk 1]
```diff
@@ -117,6 +117,12 @@ function Pane(id) {
   cloneButton.innerHTML = 'o';
   cloneButton.title = 'disconnect';
 
+  var saveButton = document.createElement('a');
+  saveButton.innerHTML = '&#10515;';
```
Comment (szym) [inline static/panes.js:121]: It'd be nice to have something easier to read. The arrow is rather thin and hard to tell apart.

File: static/panes.js [hunk 2]
```diff
@@ -117,6 +117,12 @@ function Pane(id) {
...
+  saveButton.title = 'save';
+  saveButton.download = 'untitled_image.jpg';
+  saveButton.href = '';
```
Comment (szym) [inline static/panes.js:124]: We should make it show up only if the content is savable.

**Review by szym — APPROVED** [review]
Thanks for this.

File: static/panes.js (hunk 2 above)
Comment (ili3p) [reply static/panes.js:124]: Oh, I wasn't aware that the content can be unsavable. When can this happen? I thought the panes only show up for images and charts.

File: static/panes.js (hunk 1 above)
Comment (ili3p) [reply static/panes.js:121]: How about http://www.fileformat.info/info/unicode/char/2b07/index.htm?

**Pull request merged**